src/modules/fetchingData.js

```diff
@@ -0,0 +1,38 @@
+const getScores = async (url) => {
+  try {
+    const response = await fetch(url);
+    if (!response.ok) {
+      return new Error(response.statusText);
+    }
+
+    const data = await response.json();
+
+    return data;
+  } catch (error) {
+    return error;
+  }
+};
+
+const postScore = async (url, userData) => {
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+      body: JSON.stringify(userData),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    });
+
+    if (!response.ok) {
+      return new Error(response.statusText);
+    }
```

Comment:
- [ ] [Optional] There is no reason to add this condition here because you already have the `catch` method below for that, so kindly remove it.